references/plugins/references.go

```diff
@@ -544,6 +547,31 @@ type ReferenceParameterTable struct {
 	Depth      *int
 }
 
+// ParseLocalDefinition stores the information of the definition parsed from local file
+type ParseLocalDefinition struct {
+	APIVersion string `yaml:"apiVersion"`
+	Kind       string `yaml:"kind"`
+	Metadata   struct {
+		Name      string `yaml:"name"`
+		Namespace string `yaml:"namespace"`
+		Labels    struct {
+			Type string `yaml:"type"`
+		} `yaml:"labels"`
```

Comment:
It's absolutely righ to define a struct. But we recommend to unmarshal the definition to a ComponentDefinition `https://github.com/oam-dev/kubevela/blob/master/apis/core.oam.dev/v1beta1/componentdefinition_types.go#L81`.
After that, you got a ComponentDefinition object as the way from a remote cluster https://github.com/oam-dev/kubevela/blob/master/references/plugins/cluster.go#L306-L315.

Reply:
Sure, I'll change the ParseLocalFile so it will unmarshal the local file to ComponentDefinition first, then convert it to Capability.